app/templates/views/service-settings/name-local.html

```diff
@@ -27,11 +27,11 @@
 
   <div class="form-group">
     {% if current_service.prefix_sms %}
-      <p class="govuk-body">Users will see your service name:</p>
-      <ul class="govuk-list govuk-list--bullet">
+      <p class="govuk-body">Users will see your service name at the start of every text message</p>
+      <!-- <ul class="govuk-list govuk-list--bullet">
         <li>at the start of every text message</li>
         <li>as your email sender name</li>
-      </ul>
+      </ul> -->
     {% else %}
       <p class="govuk-body">Users will see your service name as your email sender name.</p>
```

Comment:
Should we hide this sentence while we're at it?

Reply:
No because we still wanted to keep the top bullet, is why it's one sentence now instead of bullets

Reply:
No, because I pulled the text from the top bullet point to the sentence per Tim's feedback